examples/starknet-react-next/.env.development

```diff
@@ -1,3 +1,2 @@
-NEXT_PUBLIC_RPC_MAINNET="http://localhost:8001/x/starknet/mainnet"
-NEXT_PUBLIC_RPC_SEPOLIA="http://localhost:8001/x/starknet/sepolia"
```

Comment:
8001 🤔  I guess it wasn't really used...

Well, only token page is using provider from starknet-react it seems.